x11/xshm.nim

```diff
@@ -2,8 +2,8 @@
 import
   x, xlib
 
-#const
-#  libX11* = "libX11.so"
+const
+ libXext* = "libXext.so"
```

Comment:
this is gonna work only for `libXext-dev`-like packages on ubuntu and similiar distros. you should use `libXext.so(.6|)` to also match `libXext.so.6`